server/sonar-server/src/test/java/org/sonar/server/organization/DefaultOrganizationProviderRule.java

```diff
@@ -19,52 +19,24 @@
  */
 package org.sonar.server.organization;
 
-import com.google.common.base.Preconditions;
 import org.junit.rules.ExternalResource;
-import org.sonar.core.util.UuidFactoryImpl;
+import org.sonar.db.DbTester;
 
 public class DefaultOrganizationProviderRule extends ExternalResource implements DefaultOrganizationProvider {
```

Comment:
No problem for renaming this class `DefaultOrganizationDbProvider` and to bring back `DefaultOrganizationProviderRule`, but still not in shame IMO as it's never used without DbTester in the current state of codebase.